frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -107,77 +94,167 @@ const DashboardInfoSidebarBody = ({
-          isDisabled={!canWrite}
-          onChange={handleDescriptionChange}
-          onFocus={() => setDescriptionError("")}
-          onBlur={handleDescriptionBlur}
-          isOptional
-          isMultiline
-          isMarkdown
-          hasError={!!descriptionError}
-          placeholder={t`Add description`}
-          key={`dashboard-description-${dashboard.description}`}
-          style={{ fontSize: ".875rem" }}
-        />
+    <Stack spacing="lg">
+      <SidesheetCard title={t`Description`} pb="md">
+        <div className={DashboardInfoSidebarS.EditableTextContainer}>
+          <EditableText
+            initialValue={dashboard.description}
+            isDisabled={!canWrite}
+            onChange={handleDescriptionChange}
+            onFocus={() => setDescriptionError("")}
+            onBlur={handleDescriptionBlur}
+            isOptional
+            isMultiline
+            isMarkdown
+            placeholder={t`Add description`}
+          />
+        </div>
         {!!descriptionError && (
           <Text color="error" size="xs" mt="xs">
             {descriptionError}
           </Text>
         )}
-      </ContentSection>
+      </SidesheetCard>
 
       {!dashboard.archived && (
-        <ContentSection>
-          <Stack spacing="md">
-            <Switch
-              disabled={!canWrite}
-              label={t`Auto-apply filters`}
-              labelPosition="left"
-              variant="stretch"
-              size="sm"
-              id={autoApplyFilterToggleId}
-              checked={dashboard.auto_apply_filters}
-              onChange={e => handleToggleAutoApplyFilters(e.target.checked)}
-            />
-            {showCaching && isCacheable && (
-              <PLUGIN_CACHING.SidebarCacheSection
-                model="dashboard"
-                item={dashboard}
-                setPage={setPage}
-              />
-            )}
-          </Stack>
-        </ContentSection>
+        <SidesheetCard>
+          <Switch
+            disabled={!canWrite}
+            label={t`Auto-apply filters`}
+            labelPosition="left"
+            variant="stretch"
+            size="sm"
+            id={autoApplyFilterToggleId}
+            checked={dashboard.auto_apply_filters}
+            onChange={e => handleToggleAutoApplyFilters(e.target.checked)}
+          />
+        </SidesheetCard>
       )}
 
-      <ContentSection>
-        <HistoryHeader>{t`History`}</HistoryHeader>
-        <Timeline
-          events={getTimelineEvents({ revisions, currentUser })}
-          data-testid="dashboard-history-list"
-          revert={revision => dispatch(revertToRevision(revision))}
-          canWrite={canWrite}
-        />
-      </ContentSection>
-    </>
+      {showCaching && isCacheable && (
+        <SidesheetCard title={t`Caching`} pb="md">
+          <PLUGIN_CACHING.SidebarCacheSection
+            model="dashboard"
+            item={dashboard}
+            setPage={setPage}
+          />
```

Comment:
As mentioned, simplifying down to one SidebarCacheSection component seems smart. Thanks for doing that.

I'm seeing this when I run this code:

![image](https://github.com/user-attachments/assets/de4a2f45-50e8-423e-97a4-7e4cfa02cdfd)

Looks like figma has "When to get new results" and a chevron:

![image](https://github.com/user-attachments/assets/f2680b8d-15b4-4956-af01-57c2181ed421)

Reply:
@vbenedetti:

To my mind, the design in Figma works because the policy name is close enough to the rest of the text that their hierarchical relationship is clear. It's less clear to me when the sidesheet is wider.

Re "When to get new results": Framing cache deletion in terms of 'new results' stresses (if I can put it this way) birth rather than death, which is more upbeat, but I wonder if the phrase might confuse a newcomer. If I visit a dashboard once a day, and its caching policy is "hourly", I won't really get new results hourly. What's really happening is that the cache is cleared hourly. So this "new results" framing might make someone think that the caching policy could cause the dashboard to auto-refresh. Another thing: "When to get new results" could equal "Hourly", but I'm not sure it could equal "Adaptive" or "Inherit".

Some possible alternative phrases:

* Caching policy: Clear cache hourly
* Caching policy: Clear cache daily
* Caching policy: Inherit
* Caching policy: Adaptive

The last one is unclear at first, but I think the idea is that you can click on it to find out what it means, by looking at the radio button in the larger form.

(By the way, strictly speaking the cache is _invalidated_ - it is not always truly _deleted_ - but I'm assuming we can overlook the difference for the sake of simplicity.)